scripts/nesi_scripts/vm_params_gen.sl

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# script version: slurm
+#
+# must be run with sbatch vm_gen.sl [realisationCSV] [OutputDirectory] [VM_VERSION] [VM_TOPO] [HH] [PGV_THRESHOLD] [DS_MULTIPLIER] [MGMT_DB_LOC] [REL_NAME] [managementDBLocation]
+
+#SBATCH --job-name=VM_PARAMS
+#SBATCH --time=00:15:00
+#SBATCH --cpus-per-task=1
+
+if [[ -n ${CUR_ENV} && ${CUR_HPC} != "mahuika" ]]; then
+    source $CUR_ENV/workflow/install_workflow/helper_functions/activate_env.sh $CUR_ENV "mahuika"
+fi
+
+REL_CSV=$1
```

Comment:
with this number of arguments required, i would not only suggest having a empty-value check, but also put down a 

if [[ ! $# != 9 ]]

to make sure the numbers are correct.